rpc/src/eth_api.rs

```diff
@@ -119,8 +119,8 @@ pub trait EthApi {
     #[method(name = "eth_getTransactionCount")]
     async fn transaction_count(
         &self,
-        address: Address,
```

Comment:
Why did you add the `_` for address and block_number? In rust, you prefix your variable name in order to inform the compiler that you are aware of the unused variable. But in our case address and block_number are used.

Reply:
oh i see, gotcha will remove it